Foxtrot/Driver/Rewriting/EmitAsyncClosure.cs

```diff
@@ -59,6 +58,121 @@ namespace Microsoft.Contracts.Foxtrot
     /// </remarks>
     internal class EmitAsyncClosure : StandardVisitor
     {
+        /// <summary>
+        /// Class that maps generic arguments of the enclosed class/method the generic arguments of the closure.
+        /// </summary>
+        /// <remarks>
+        /// The problem.
+        /// Original implementation of the Code Contract didn't support async postconditions in generics.
+        /// Here is why:
+        /// Suppose we have following function (in class <code>Foo</code>:
```

Comment:
To inline code into a text a <c>Foo</c> form may be used. <code> will move its content to a new line.